tests/saw/sike_r2/proof/field_specs.saw

```diff
@@ -0,0 +1,249 @@
+// field_common.saw Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: filename in comment (`field_common.saw`) doesn't match actual filename (`field_specs.saw`)